packages/react-dom/src/__tests__/ReactTestUtilsAct-test.js

```diff
@@ -371,18 +370,23 @@ describe('ReactTestUtils.act()', () => {
       let ctr = 0;
       const div = document.createElement('div');
 
-      await act(async () => {
-        act(() => {
-          ReactDOM.render(<App callback={() => ctr++} />, div);
-        });
-        expect(div.innerHTML).toBe('0');
-        expect(ctr).toBe(1);
+      act(() => {
+        ReactDOM.render(<App callback={() => ctr++} />, div);
       });
-      // this may seem odd, but it matches user behaviour -
-      // a flash of "0" followed by "1"
+      expect(div.innerHTML).toBe('0');
+      expect(ctr).toBe(1);
+
+      await act(async () => {}); // #just-react-things
```

Comment:
Why did you remove the `await` from the earlier `act`?

Reply:
I wanted to specifically test the intermediate sync state. this test is named badly (and frankly on looking at it now, it doesn't seem super useful).

Reply:
It's not clear to me at all what behavior you're actually trying to test. More comments could help.